includes/admin/admin-actions.php

```diff
@@ -76,10 +76,12 @@ function give_redirect_to_clean_url_admin_pages() {
 	if ( $redirect ) {
 		// Redirect.
 		wp_redirect(
-			remove_query_arg(
-				[ '_wp_http_referer', '_wpnonce' ],
-				wp_unslash( $_SERVER['REQUEST_URI'] )
-			)
+            esc_url(
```

Comment:
@JasonTheAdams WordPress suggested using esc_url_raw when the URL is part of the HTTP header.

ref: https://make.wordpress.org/plugins/2015/04/20/fixing-add_query_arg-and-remove_query_arg-usage/

> When it’s being printed to a page (for example as a link), you should use [esc_url()](https://developer.wordpress.org/reference/functions/esc_url/). When it’s being used in HTTP headers or as part of a HTTP request (for example, as part of a location redirect header or in a [wp_remote_get()](https://developer.wordpress.org/reference/functions/wp_remote_get/) call), you should use [esc_url_raw()](https://developer.wordpress.org/reference/functions/esc_url_raw/).

Reply:
Good catch! I didn't think about the redirect functions. Resolved in 2877aba0